shared/js/url_test/url_test.js

```diff
@@ -2,14 +2,7 @@
 // Useful as a workaround for CORS security to test access to an origin.
 function testImageAccess(url, successCallback, failureCallback, timeoutMs) {
   timeoutMs = typeof timeoutMs !== 'undefined' ?  timeoutMs : 5000;
-  var img = document.createElement('img');
-  img.width = 1;
-  img.height = 1;
-  img.style.position = 'absolute';
-  img.style.bottom = 0;
-  img.style.width = 1;
-  img.style.height = 1;
-  img.style.visibility = 'hidden';
+  var img = new Image();
```

Comment:
From the usage examples I found, it didn't seem like actually adding the image to the DOM was a requirement, so this change simplifies the logic in the class, reducing the surface area for subtle cross-browser differences to emerge.